### PR TITLE
Add style

### PR DESCRIPTION
**Pull request by vladanvi99**
In this pull request, I added style to my elements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,26 @@
...
+## Setup
+
+### Clone this repository
+
+```bash
+$ git clone https://github.com/vladanvi99/leaderboard.git
+$ cd leaderboard
+```
+
+### Run project
```
Comment (vichuge) [inline README.md:20]: - [ ] On `Run project` section, you could add the way to use the project with `npm start`. 🐶 
- [ ] Please, add instructions about how to use the API (how to create your own game, how to use the API, credentials if is required, etc.). 🐱

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, consider:
- [OPTIONAL]

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -21,12 +21,50 @@ $ cd leaderboard
...
+ fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/idOfGame/scores/', {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ user: "name Of user", score: "score of user" }),
+    });
+
+```
```
Comment (mricanho) [inline README.md:67]: - [ ] [OPTIONAL] You can omit this part, remember that the README file is for everyone, this means developers and non-developers, the last Code Reviewer was asking for special instructions about the end-points or if the API needed some kind of key. But he didn't explain himself very well, so don't worry.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @vladanvi99  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

yougotit

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (vladanvi99) [reply README.md:67]: I understand, thank you very much. 😃

**Pull request merged**